targets/TARGET_NXP/TARGET_LPC408X/device/system_LPC407x_8x_177x_8x.c

```diff
@@ -555,7 +555,8 @@ fpu_init();
   LPC_SC->CLKOUTCFG = CLKOUTCFG_Val;    /* Clock Output Configuration         */
 #endif
 
-  LPC_SC->PBOOST 	|= 0x03;			/* Power Boost control				*/
+  //TODO: verify correct operation w/ this setting. Refer to Table 18 in UM10562.pdf.
```

Comment:
How this line got in? Was it bad rebase? I don't see this todo. And this PR was updated not to change targets/ code ?

Reply:
Good catch; my mistake! That's from unrelated work, which was not meant to be published.

I added a 4th commit, to undo the bad part from the 3rd commit. Should I squash the 4th into the 3rd?